xsd/NeTEx_publication.xsd

```diff
@@ -5268,49 +5467,10 @@ Provides a general purpose wrapper for NeTEx data content.</xsd:documentation>
 			<xsd:field xpath="@version"/>
 		</xsd:keyref>
 		<xsd:key name="ServicePattern_AnyVersionedKey">
-			<xsd:selector xpath=".//netex:ServicePattern |.//netex:DeadRunPattern "/>
+			<xsd:selector xpath=".//netex:ServicePattern |.//netex:DeadRunPattern"/>
 			<xsd:field xpath="@id"/>
 			<xsd:field xpath="@version"/>
 		</xsd:key>
-		<!-- =====PointInJourneyPatternLink============================== -->
```

Comment:
Just wondering where did go to...

Reply:
Moved to NeTEx_publication_timetable.xsd:5316
(diff v1.0 vs v1.1 are similar, but the concept is removed from xsd/NeTEx_publication_timetable.xsd in v1.1)